fhd_core/calibration/vis_calibrate.pro

```diff
@@ -168,7 +168,7 @@ FOR iter=0,calibration_flag_iterate DO BEGIN
     t2+=t3_a-t2_a
     
     IF Keyword_Set(flag_calibration) THEN vis_calibration_flag,obs,cal,n_tile_cut=n_tile_cut,_Extra=extra
-    IF n_tile_cut EQ 0 THEN BREAK
+
```

Comment:
Ah! You are right, obvious in retrospect. Since in-situ sim skips a flagging procedure (vis_flag_basic), n_tile_cut isn't defined and breaks on this line. Will change to keyword_set.